tests/python/twitter/common/reviewboard/test_reviewboard.py

```diff
@@ -0,0 +1,22 @@
+# ==================================================================================================
+# Copyright 2011 Twitter, Inc.
+# --------------------------------------------------------------------------------------------------
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this work except in compliance with the License.
+# You may obtain a copy of the License in the LICENSE file, or at:
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==================================================================================================
+
+import pytest
+from twitter.common.reviewboard import ReviewBoard
```

Comment:
It should be
```
from twitter.common.reviewboard import ReviewBoard

import pytest
```

* That is, import reorder and a newline between first and the second.

With these two comments addressed, you could merge it in. #shipit